addon/poll-task.js

```diff
@@ -158,15 +158,32 @@ export function pollTask(obj, taskOrName, token = getNextToken()) {
    @param { String } token the token for the pollTask to be cleared
    @public
    */
-export function cancelPoll(token) {
+export function cancelPoll(obj, token) {
+  // TODO: remove this older API on next major version
```

Comment:
I don't think we need the TODO since this also adds the deprecations (all deprecations are TODO's for a major version bump)...